docs/source/guides/serializers.rst

```diff
@@ -0,0 +1,48 @@
+Serializers -- saving and loading
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Serializer is a simple interface to serialize or deserialize an object.
+:class:`Link`, :class:`Optimizer`, and :class:`~training.Trainer` supports serialization.
+
+Concrete serializers are defined in the :mod:`serializers` module.
+It supports NumPy NPZ and HDF5 formats.
+
+For example, we can serialize a link object into NPZ file by the :func:`serializers.save_npz` function:
+
+.. doctest::
+
+   >>> from chainer import serializers
+   >>> serializers.save_npz('my.model', model)
```

Comment:
It seems like the variable `model` here is undefined.

Reply:
Added explanation.